pkg/kernel/kernel_config-4.9.x-x86_64

```diff
@@ -808,10 +808,10 @@ CONFIG_NET_IPVTI=m
 CONFIG_NET_UDP_TUNNEL=y
 CONFIG_NET_FOU=y
 CONFIG_NET_FOU_IP_TUNNELS=y
-CONFIG_INET_AH=y
-CONFIG_INET_ESP=y
-CONFIG_INET_IPCOMP=y
-CONFIG_INET_XFRM_TUNNEL=y
+CONFIG_INET_AH=m
+CONFIG_INET_ESP=m
+CONFIG_INET_IPCOMP=m
+CONFIG_INET_XFRM_TUNNEL=m
```

Comment:
so what exactly is the problem with compiling them statically?

Reply:
I am closing this pull request. These mod probe warning does not affect the functionality.